src/pages/Embeds/Plio.vue

```diff
@@ -558,10 +558,20 @@ export default {
         this.isAspectRatioChecked = true;
       }
     },
```

Comment:
the scorecard cup icon is still shown on the video slider even if all questions are survey mode questions